Sources/ImperialDropbox/DropboxRouter.swift

```diff
@@ -0,0 +1,40 @@
+import Vapor
+import Foundation
+
+public class DropboxRouter: FederatedServiceRouter {
+    public let tokens: FederatedServiceTokens
+    public let callbackCompletion: (Request, String) throws -> (EventLoopFuture<ResponseEncodable>)
+    public var scope: [String] = []
+    public let callbackURL: String
+    public let accessTokenURL: String = "https://api.dropboxapi.com/oauth2/token"
+    
+    public var callbackHeaders: HTTPHeaders {
+        ["Content-Type": HTTPMediaType.urlEncodedForm.description,
+         "Authorization": "Basic \(encodedClientCredentials)"]
+    }
+    
+    private var encodedClientCredentials: String {
+        Data("\(tokens.clientID):\(tokens.clientSecret)".utf8).base64EncodedString()
+    }
```

Comment:
Consider rewriting these properties this way:
```suggestion
    public var callbackHeaders: HTTPHeaders {
        var headers: HTTPHeaders
        headers.basicAuthorization = .init(username: tokens.clientID, password: tokens.clientSecret)
        headers.contentType = .urlEncodedForm
        return headers
    }
```
This way, you neither have to reimplement the use of Basic authorization (such as it is!) nor lose any fidelity the `HTTPMediaType` may provide beyond just the MIME type (not really relevant for `.urlEncodedForm`, but it's a good habit to be in just the same).